web/src/cypress/tsconfig.json

```diff
@@ -4,7 +4,19 @@
     "baseUrl": "../node_modules",
     "target": "es5",
     "lib": ["es2015", "dom"],
-    "types": ["cypress"]
+    "types": ["cypress"],
+    "jsx": "react",
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    // TODO: set to noImplicitAny to true
+    "noImplicitAny": true,
```

Comment:
rm comment?

Reply:
It looks like it's already set to `true` though?

Reply:
that was a typo, meant to be `isolatedModules` not `noImplicitAny`. PR here for that fix: #532